chips/arty_e21/src/chip.rs

```diff
@@ -114,19 +119,19 @@ impl ArtyExx {
     /// operations. Different boards can call the functions that `initialize()`
     /// calls directly if it needs to use a custom setup operation.
     pub unsafe fn initialize(&self) {
-        self.disable_pmp();
+        self.mpu().disable_mpu();
```

Comment:
I think we can remove the `disable_pmp()` function above.

Reply:
:+1: done